easybuild/easyconfigs/c/cuDNN/cuDNN-5.1-CUDA-8.0.44.eb

```diff
@@ -24,7 +24,7 @@ sources = ['%(namelower)s-8.0-linux-x64-v%(version)s.tgz']
 dependencies = [('CUDA', cuda_version)]
 
 checksums = [
-    '099ca6a9b21b35e32f3408eff2a1c06d',    # cudnn-8.0-linux-x64-v5.1.tgz
+    '406f4ac7f7ee8aa9e41304c143461a69',    # cudnn-8.0-linux-x64-v5.1.tgz, Jan 20 2017 download
```

Comment:
urgh, indeed, redownloading now gives me the same MD5 as you, great...

I wonder why @NVIDIA would just update tarballs in place without a proper version bump?